recipes/cmor/meta.yaml

```diff
@@ -0,0 +1,33 @@
+package:
+    name: cmor
+    version: 3.2.7
+
+source:
+    git_rev: master
+    git_url: git://github.com/PCMDI/cmor.git
```

Comment:
I also wonder about the `git://` vs `https://` which is what is noted in the documentation https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#source-from-git

@dnadeau4 @doutriaux1

Reply:
actually it probably ought to be: https://pcmdi.github.io/cmor-site/

Reply:
@doutriaux1 you're getting the `source` vs `about` entries confused.

`source` should be `https://` and `about` should be https://cmor.llnl.gov/

Reply:
@durack1 yes typed reply in wrong box. I meant for the "home" entry